Probe/logger.go

```diff
@@ -0,0 +1,37 @@
+/*
+ *  Copyright 2020 Google LLC
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package main
+
+import "log"
+
+type logger interface {
+	logProbe(tim string, st string, lat int)
+}
+
+type cloudLogger struct {
+	// Will contain items when logging is implemented
```

Comment:
I would make these TODOs

// TODO(robparker): Implement this when cloud logging is setup
// TODO(b/1232323): ...

Reply:
addressed in a8aedf9